pom.xml

```diff
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>org.apache.sshd</groupId>
       <artifactId>sshd-core</artifactId>
-      <version>1.7.0</version>
+      <version>2.6.0</version>
```

Comment:
Is there a risk that plugin functionality is affected by the same issue as https://github.com/jenkinsci/jenkins/pull/5506 or is it a specific issue of the CLI?

Reply:
On my test, I did not found any issue, but I will wait for 2.7.0 that contains the fix just in case

Reply:
indeed 2.7.0 is released so I will bump it to 2.7.0

Reply:
I thought about it and finally, I have made another PR for bump to 2.5.1 https://github.com/jenkinsci/sshd-plugin/pull/46 , when the 2.7.0 I will rework in this one, but let's move forward and make the bump.

Reply:
I think 2.7.0 is out, so can this PR be readied for review?

Reply:
also https://github.com/jenkinsci/jenkins/pull/5547

Reply:
```suggestion
      <version>2.7.0</version>
```